Assets/Packages/com.studio23.ss2.splashscreensystem/Runtime/Core/SplashScreenBehaviour.cs

```diff
@@ -3,14 +3,19 @@
 using Studio23.SS2.SplashScreenSystem.UI;
 using System;
 using UnityEngine;
+using UnityEngine.Events;
 
 namespace Studio23.SS2.SplashScreenSystem.Core
 {
+    [System.Serializable]
+    public class UnityEventWithBool : UnityEvent<bool> {}
```

Comment:
No need for explicitly inherit generic unityevent.

Reply:
There are no bool return with generic unity event